src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,779 @@
+all the things needed to actually build the Preference Editor.
+The Auxiliary Schema for our example Editor
+is defined in the `schemas/auxiliary.js` file:
+
+```javascript
+fluid.defaults("minEditor.auxSchema", {
+    gradeNames: ["fluid.prefs.auxSchema"],
+    auxiliarySchema: {
+          // some code not shown
+    }
+});
+```
+Again, we use `fluid.defaults()` to create the Schema.
+As with the Primary Schema and the Panel, `fluid.defaults()` is passed two arguments:
+1) a string name (`"minEditor.auxSchema"`), and 2) a JavaScript object literal
+containing configuration options.
+
+Let’s look at the Schema itself in detail:
+```javascript
+auxiliarySchema: {
+    // the loaderGrade identifies the "base" form of preference editor desired
+    loaderGrades: ["fluid.prefs.fullNoPreview"],
+
+    // 'terms' are strings that can be re-used elsewhere in this schema;
+    terms: {
+        templatePrefix: "html"
+    },
+
+    // the main template for the preference editor itself
+    template: "%templatePrefix/minEditor.html",
+
+    autoPilot: {
+        // this 'type' must match the name of the pref in the primary schema
+        type: "minEditor.autoPilot",
+        panel: {
+            // this 'type' must match the name of the panel grade created for this pref
+            type: "minEditor.panels.autoPilot",
+
+            // selector indicating where, in the main template, to place this panel
+            container: ".mec-autoPilot",
+
+            // the template for this panel
+            template: "%templatePrefix/autoPilot.html"
+        }
+    }
+}
+});
+```
+An auxiliary schema can be generally divided into two types of properties:
+
+1. top-level members, defining globally-used values, and
+2. per-preference members (one per preference), defining the specific requirements for each preference.
+
+
+##### Loader Grade #####
+The _loader grade_ specifies the _type_ of Preference Editor:
+
+```javascript
+    loaderGrades: ["fluid.prefs.fullNoPreview"],
+```
+
+The Preference Framework provides three pre-defined types of Editor:
+1. separated panel (the default): This is a page-width panel collapsed at the top of the page;
+it slides down when activated by the user.
+2. full page, no preview: This is a Preference Editor that occupies the full page.
+3. full page, with preview: This is a Preference Editor that occupies the full page, but includes
+provisions for an iframe in the page to preview any changes made by the Editor.
+
+In the code snippet above, the `loaderGrades` option is used to specify the “full page, no preview” form.
+
+##### Templates #####
+The Auxiliary Schema must declare where to find the main HTML template for the Preference Editor.
+In our example, this template is located in the same folder as other HTML templates.
+The Auxiliary Schema allows you to define `terms` – strings that can be re-used elsewhere in the
+schema. Here, it is being used to define once the path to where the HTML templates are:
```

Comment:
I'm not sure if the "once" is needed in the above sentence, or if some punctuation would help clarify things.